src/core/game.ts

```diff
@@ -31,7 +32,7 @@ export interface GameState {
 };
 
 export class Game {
-  static settings = { minPlayers: 2, maxPlayers: 4 };
+  static settings = { minPlayers: 2, maxPlayers: 2 };
```

Comment:
I'd prefer you switch this back to max 4 since we don't want to break the <max players use case